.travis.yml

```diff
@@ -373,3 +398,13 @@ script:
       sudo cp test-launcher.sh rootfs;
       travis_retry ${B2} arm-tests warnings-as-errors=on warnings=all crypto=$crypto dht=off extensions=off logging=off deprecated-functions=off invariant-checks=off export-extra=on asserts=on $toolset target-os=linux link=static testing.launcher="sudo chroot rootfs /bin/sh test-launcher.sh";
     fi'
+  - 'if [[ "$arch" == "s390x" ]];
+    then
+      ${B2} s390x-tests warnings-as-errors=on warnings=all crypto=$crypto dht=on extensions=off logging=off deprecated-functions=off invariant-checks=off export-extra=on asserts=on $toolset target-os=linux link=static testing.execute=off;
```

Comment:
I can't remember, but is there a good reason to disable invariant-checks logging and extensions? Is the idea that the emulation will be so slow that we want to do as little as possible?

Reply:
yes, the idea is to do as little as possible and I doubt any "live network" test will work reliable, however that was the case for arm and I just copy-pasted :)